locale/af/LC_MESSAGES/django.po

```diff
@@ -1116,8 +1116,9 @@ msgstr ""
 
 #. This is a header of a list of things the add-on can do.
 #: src/olympia/addons/templates/addons/impala/details.html
-msgid "It can:"
-msgstr ""
+#, fuzzy
+msgid "This add-on can:"
+msgstr "Dié byvoeging het 'n privaatheidbeleid"
```

Comment:
This string is marked as fuzzy so it's probably not the localisers fault but the extraction/merger's so it needs to be fixed by the localiser